guidance_node:
  ros__parameters:
    use_nav_odom: true          
    setpoint_topic: "/guidance/setpoint"
    nav_odom_topic: "/nav/odom"
    sim_odom_topic: "/sim/odom"

    rate_hz: 20.0
    z_ref: 1.0

    # 모드 설정 파라미터 ("lookahead" or "min_jerk" or "min_snap" or "multi_snap")
    guidance_mode: "multi_snap" 

    # waypoints_x: [0.0, 5.0, 5.0, 0.0, 0.0]
    # waypoints_y: [0.0, 0.0, 5.0, 5.0, 0.0]
    # 뾰족한 모서리를 잘라내고 진입/진출점을 명시하여 '둥근 사각형' 유도
    waypoints_x: [0.0, 4.0, 5.0, 5.0, 4.0, 1.0, 0.0, 0.0]
    waypoints_y: [0.0, 0.0, 1.0, 4.0, 5.0, 5.0, 4.0, 0.0]

    # 6주차 (Look-ahead) 용 파라미터
    accept_radius: 0.5          
    lookahead_dist: 1.5         

    # 7주차 (Polynomial) 용 파라미터
    avg_speed: 1.5              
    
    hold_last: true